### PR TITLE
Added SignUp And Login

### PR DESCRIPTION
**Pull request by SimranjitKaurMaan**
Added SignUp and Login Pages
image

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                id="search-bar"
+                placeholder="Search"
+              />
+            </form>
+            </div>
+            <button class="btn btn-primary">Login</button>
+            <div class="header-nav">
+                <nav>
+                    <ul>
+                        <li>
```
Comment (shubhamkumar648) [inline index.html:31]: avoid space

Comment (SimranjitKaurMaan) [reply index.html:31]: I used prettify for indentation. I think for readability space is added.

Comment (shubhamkumar648) [reply index.html:31]: ok

**Review by shubhamkumar648 — APPROVED** [review]
> Remaing code good and clean . you can merge

Thanks for reviewing this.

**Review by ttahm3d — APPROVED** [review]
Can be merged.

**Pull request merged**